2024-12-24 19:39:33,778 [WARNING] Not Found: /serviceworker.js
2024-12-24 19:39:35,462 [WARNING] Not Found: /serviceworker.js
2024-12-24 19:39:42,118 [INFO] /app/app/views.py changed, reloading.
2024-12-24 19:40:01,013 [WARNING] Not Found: /serviceworker.js
2024-12-24 19:41:34,963 [INFO] /app/app/views.py changed, reloading.
2024-12-24 19:41:44,842 [INFO] /app/app/views.py changed, reloading.
2024-12-24 19:41:49,231 [INFO] /app/app/views.py changed, reloading.
2024-12-24 19:43:52,469 [INFO] /app/app/views.py changed, reloading.
2024-12-24 19:43:57,139 [WARNING] Not Found: /serviceworker.js
2024-12-24 19:44:02,358 [INFO] Port scan suspects: {}
2024-12-24 19:44:02,962 [ERROR] Error generating traffic graph: [Errno 2] No such file or directory: '/app/static/traffic_graph.png'
Traceback (most recent call last):
  File "/app/app/views.py", line 161, in generate_traffic_graph
    plt.savefig(graph_path)
  File "/usr/local/lib/python3.11/site-packages/matplotlib/pyplot.py", line 1243, in savefig
    res = fig.savefig(*args, **kwargs)  # type: ignore[func-returns-value]
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/matplotlib/figure.py", line 3490, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/usr/local/lib/python3.11/site-packages/matplotlib/backend_bases.py", line 2184, in print_figure
    result = print_method(
             ^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/matplotlib/backend_bases.py", line 2040, in <lambda>
    print_method = functools.wraps(meth)(lambda *args, **kwargs: meth(
                                                                 ^^^^^
  File "/usr/local/lib/python3.11/site-packages/matplotlib/backends/backend_agg.py", line 481, in print_png
    self._print_pil(filename_or_obj, "png", pil_kwargs, metadata)
  File "/usr/local/lib/python3.11/site-packages/matplotlib/backends/backend_agg.py", line 430, in _print_pil
    mpl.image.imsave(
  File "/usr/local/lib/python3.11/site-packages/matplotlib/image.py", line 1634, in imsave
    image.save(fname, **pil_kwargs)
  File "/usr/local/lib/python3.11/site-packages/PIL/Image.py", line 2600, in save
    fp = builtins.open(filename, "w+b")
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/app/static/traffic_graph.png'
2024-12-24 19:46:56,596 [INFO] /app/app/views.py changed, reloading.
